src/plugins/toggle/toggle-en.hbs

```diff
@@ -20,17 +20,18 @@
 
 <div class="row">
 
-	<div class="col-md-3">
-		<div class="list-group">
-			<a href="#setup" class="list-group-item">Plugin Setup</a>
-			<a href="#simple" class="list-group-item">Simple Example</a>
-			<a href="#details" class="list-group-item">Details Elements</a>
-			<a href="#print" class="list-group-item">Print States</a>
-			<a href="#grouped" class="list-group-item">Grouped Toggle</a>
-			<a href="#accordion" class="list-group-item">Accordion</a>
-			<a href="#persist" class="list-group-item">Persist Toggle State</a>
-		</div>
-	</div>
+	<nav class="col-md-3">
+		<h2 class="wb-inv">Table of contents</h2>
+		<ul class="list-group list-unstyled">
+			<li><a href="#setup" class="list-group-item">Plugin Setup</a></li>
+			<li><a href="#simple" class="list-group-item">Simple Example</a></li>
+			<li><a href="#details" class="list-group-item">Details Elements</a></li>
+			<li><a href="#print" class="list-group-item">Print States</a></li>
+			<li><a href="#grouped" class="list-group-item">Grouped Toggle</a></li>
+			<li><a href="#accordion" class="list-group-item">Accordion</a></li>
+			<li><a href="#persist" class="list-group-item">Persist Toggle State</a></li>
```

Comment:
How about we put the class on the `<li>` instead of the `<a>` so that there are rounded corners only in the top and bottom of the list ?

```suggestion
			<li class="list-group-item"><a href="#setup">Plugin Setup</a></li>
			<li class="list-group-item"><a href="#simple">Simple Example</a></li>
			<li class="list-group-item"><a href="#details">Details Elements</a></li>
			<li class="list-group-item"><a href="#print">Print States</a></li>
			<li class="list-group-item"><a href="#grouped">Grouped Toggle</a></li>
			<li class="list-group-item"><a href="#accordion">Accordion</a></li>
			<li class="list-group-item"><a href="#persist">Persist Toggle State</a></li>
```

Reply:
I was thinking of doing that from the outset since it'd look nicer, but decided against it because that setup would cause the links themselves to revert to ``display: inline;`` (making them more difficult to click/touch).

FWIW the [WET theme's secondary menu demo](https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html) structured its list of links similarly to this PR. But it uses CSS overrides tied to the ``wb-sec`` ID to counteract the rounded corners.

I could potentially go with what you proposed and throw the ``stretched-link`` class onto every link, but aren't very keen about that combo. The status quo causes focus outlines to appear on the rectangle borders, whereas using the ``stretched-link`` class would cause the outlines to appear on the text (which looks kind of strange).

Something else I could do would be to keep the status quo, but throw ``brdr-rds-0`` classes onto all the links to nerf all the rounded corners.

**PS:**
Bootstrap never improved their reference list group link examples overtime :(. [Bootstrap 5.3 is still using a list-less group of links](https://getbootstrap.com/docs/5.3/components/list-group/#links-and-buttons) for this scenario to this day :S...

Reply:
Let's keep the UI consistent with other plugins and remove the sidebar. Instead, add a section at the top with the heading "On this page:". Just like the Paginate, Tag filter, or Utilities pages.

Reply:
Works for me, but just wanted to note that ToCs are all over the place in WET and (to a lesser extent) GCWeb.

Some pages use ["Table of contents"](https://wet-boew.github.io/v4.0-ci/demos/lightbox/lightbox-en.html), whereas others use ["On this page" (no colon)](https://wet-boew.github.io/v4.0-ci/unmin/demos/wamethod/wamethod-AA-en.html) or ["On this page:" (colon)](https://wet-boew.github.io/v4.0-ci/demos/paginate/paginate-en.html). Structure-wise, most use ``<nav>`` elements, some still use ``<nav role="navigation">`` and a couple don't use any sectioning :S.

I think "Table of contents" was the original convention for ToCs and "On this page" was a pattern DTO created for Canada.ca that PP subsequently snuck into WET.

Reply:
Done, although I went with "On this page"/"Sur cette page" (without a colon).

IMO using a colon at the end of headings looks kind of strange. Plus the [CDS' in-page table of contents design pattern](https://design.canada.ca/common-design-patterns/in-page-toc.html) doesn't advocate using a colon.

Might send another PR to normalize all ToC wording and structure at some point later on.

Reply:
Thank you @EricDunsworth ! I agree with you about the colon. Feel free to take the time to normalize all table of contents. It would be greatly appreciated. :)